dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/metrics/BatchMetricsDispatcher.kt

```diff
@@ -43,7 +42,7 @@ internal class BatchMetricsDispatcher(
             internalLogger.logMetric(
                 messageBuilder = { BATCH_DELETED_MESSAGE },
                 additionalProperties = it,
-                samplingRate = MethodCallSamplingRate.LOW.rate
+                samplingRate = 1.5f
```

Comment:
For the other sampling rates we've used values from MethodCallSamplingRate. Do we not want to add an additional value there for this rate (do we not expect 1.5f to be reused)?